pages/abi.vue

```diff
@@ -1,17 +1,17 @@
 <template>
   <div class="abi-page">
```

Comment:
root class should duplicate the filename

Reply:
discussed